postgrescontroller/README.md

```diff
@@ -0,0 +1,2 @@
+# postgres-operator
+PostgreSQL Operator generated using Operator SDK library.
```

Comment:
Add more details to the README.

Explain at high-level the operation of the Operator.